tests/functional/pv/space_reclaim/test_staggered_cronjob_scheduling.py

```
@@ -328,3 +354,787 @@ def _assert_stagger_spread(self, timestamps: dict) -> None:
+
+        """
+        pvc_objs, cronjob_map = self._create_pvcs_and_wait_for_cronjobs(
+            storageclass_factory,
+            multi_pvc_factory,
+            pod_factory,
+            num_pvcs=NUM_PVCS_LARGE,
+            schedule=FIVE_MIN_SCHEDULE,
+        )
+
+        uids = set()
+        for pvc_name, cj_obj in cronjob_map.items():
```

Ruff (0.15.13)

[warning] Loop control variable `pvc_name` not used within loop body

Rename unused `pvc_name` to `_pvc_name`

(B007)